www_src/lib/keyboard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  simulateTabPress: function () {
+    var current = document.activeElement;
+    var elements = document.getElementsByTagName('input');
+    var idx = parseInt(current.getAttribute('tabIndex'), 10) + 1;
```

Comment:
vanilla JS trick: `var idx = current.getAttribute('tabIndex') | 0 + 1`, which force-coerces to a 32 bit decimal int. Mind you, jslint *will* complain about that by default (not sure if we care =)